public/components/content-list-item/_content-list-item.scss

```
@@ -164,15 +164,21 @@
         &--ophan,
         &--created,
         &--incopy,
-        &--wordcount,
-        &--printwordcount,
+        &--wordcount {
+            @extend %fs-data-1;
+        }
+
+        &--printwordcount {
+            @extend %fs-data-1;
+        }
+
```

This can be simplified to the following as it looks like all the columns now share the same `@extend %fs-data-1` styles?

```suggestion
        &--wordcount,
        &--printwordcount
```